Backpack/Tests/UnitTests/BPKColorTest.m

```diff
@@ -27,7 +27,7 @@ @interface BPKColorTest : XCTestCase
 
 @implementation BPKColorTest
 
-- (void)testColorSubsetExists {
+- (void)disabled_testColorSubsetExists {
```

Comment:
should we remove those instead or do we need to rewrite them with semantic examples?

Reply:
I have removed the tests and created a ticket to follow-up on creating new colour related tests.